docs/gas-schedule-proposal.md

```diff
@@ -0,0 +1,223 @@
+| 08 | MULMOD | 8 | 14 |
+| 0A | EXP | static_gas = 10 <br/> dynamic_gas = 50 * exponent_byte_size | static_gas = 10 <br/> dynamic_gas = 20 * exponent_byte_size |
+| 20 | KECCAK256 | static_gas = 30 <br/> dynamic_gas = 6 * minimum_word_size + memory_expansion_cost | static_gas = 50 <br/> dynamic_gas = 30 * minimum_word_size + memory_expansion_cost |
+| 30 | ADDRESS | 2 | 5 |
+| 33 | CALLER | 2 | 5 |
+| 37 | CALLDATACOPY | static_gas = 3 <br/> dynamic_gas = 3 * minimum_word_size + memory_expansion_cost | static_gas = 8 <br/> dynamic_gas = 3 * minimum_word_size + memory_expansion_cost |
+| 3E | RETURNDATACOPY | static_gas = 3 <br/> dynamic_gas = 3 * minimum_word_size + memory_expansion_cost | static_gas = 8 <br/> dynamic_gas = 3 * minimum_word_size + memory_expansion_cost |
+| 47 | SELFBALANCE | 5 | 3 |
+| 52 | MSTORE | static_gas = 3 <br/> dynamic_gas = memory_expansion_cost | static_gas = 5 <br/> dynamic_gas = memory_expansion_cost |
+| 53 | MSTORE8 | static_gas = 3 <br/> dynamic_gas = memory_expansion_cost | static_gas = 5 <br/> dynamic_gas = memory_expansion_cost |
+| 56 | JUMP | 8 | 3 |
+| 57 | JUMPI | 10 | 5 |
+| A0 | LOG0 | static_gas = 375 <br/> dynamic_gas = 375 * topic_count + 8 * size + memory_expansion_cost | static_gas = 35 <br/> dynamic_gas = 35 * topic_count + 8 * size + memory_expansion_cost |
+| A1 | LOG1 | static_gas = 375 <br/> dynamic_gas = 375 * topic_count + 8 * size + memory_expansion_cost | static_gas = 35 <br/> dynamic_gas = 35 * topic_count + 8 * size + memory_expansion_cost |
+| A2 | LOG2 | static_gas = 375 <br/> dynamic_gas = 375 * topic_count + 8 * size + memory_expansion_cost | static_gas = 35 <br/> dynamic_gas = 35 * topic_count + 8 * size + memory_expansion_cost |
+| A3 | LOG3 | static_gas = 375 <br/> dynamic_gas = 375 * topic_count + 8 * size + memory_expansion_cost | static_gas = 35 <br/> dynamic_gas = 35 * topic_count + 8 * size + memory_expansion_cost |
+| A4 | LOG4 | static_gas = 375 <br/> dynamic_gas = 375 * topic_count + 8 * size + memory_expansion_cost | static_gas = 35 <br/> dynamic_gas = 35 * topic_count + 8 * size + memory_expansion_cost |
+| F1 | CALL | address_access_cost = 100 (warm address) | address_access_cost = 150 (warm address) |
+| F4 | DELEGATECALL | address_access_cost = 100 (warm address) | address_access_cost = 150 (warm address) |
+| FA | STATICCALL | address_access_cost = 100 (warm address) | address_access_cost = 150 (warm address) |
+| F3 | RETURN | static_gas = 0 <br/> dynamic_gas = memory_expansion_cost | static_gas = 5 <br/>dynamic_gas = memory_expansion_cost |
+| FD | REVERT | static_gas = 0 <br/> dynamic_gas = memory_expansion_cost | static_gas = 8 <br/>dynamic_gas = memory_expansion_cost |
+
+
+| Precompile | Name | Current Gas | Proposed Gas |
+| ------------- | ------------- | ------------- | ------------- |
+| 01 | ECRECOVER | 3000 | 17000 |
+| 06 | ECADD | 150 | 3500 |
+| 07 | ECMUL | 6000 | 10000 |
+| 0A | POINTEVAL | 50000 | 400000 |
+
+
+## Radical Gas Schedule Proposal
```

Comment:
There were some discussion in the community on a similar approach. Maybe we could give a reference?